tests/integration.rs

```diff
@@ -3083,17 +3143,17 @@ mod tests {
 
         #[test]
         fn test_virtio_block() {
-            _test_virtio_block(FOCAL_IMAGE_NAME, false)
+            _test_virtio_block(UbuntuImageType::Focal, false, false)
         }
 
         #[test]
         fn test_virtio_block_disable_io_uring() {
-            _test_virtio_block(FOCAL_IMAGE_NAME, true)
+            _test_virtio_block(UbuntuImageType::Focal, false, true)
         }
 
         #[test]
         fn test_virtio_block_qcow2() {
-            _test_virtio_block(FOCAL_IMAGE_NAME_QCOW2, false)
+            _test_virtio_block(UbuntuImageType::FocalQcow, true, false)
```

Comment:
I would think it would be better to not have his second parameter. FocalQcow is always copied.

Reply:
Fixed!